pkg/logger/logger.go

```diff
@@ -2,6 +2,7 @@ package logger
 
 import (
 	"fmt"
+	table "github.com/MaxwelMazur/tablecli"
```

Comment:
Sorry if i am out of my "league" here, but should we link external private account resources? 
Or this could/should be some resource available inside `aziontech` domain and repositories?

Reply:
you might be right!  I think it's time we move this to aziontech (or even create a lib of our own, as the one we're using is a fork with some slight modifications)

Reply:
Hi Helio this is a lib to handle table rows and columns in a very customizable way and this is an external library made by me, and we use it in our code like so many others, thanks for the question : )

Reply:
It's also interesting to leave the package modular because it's something specific that can be used by other users